packages/nouns-contracts/tasks/populate-descriptor-v3.ts

```diff
@@ -0,0 +1,65 @@
+import { task, types } from 'hardhat/config';
+import ImageData from '../files/image-data-v2.json';
+import { dataToDescriptorInput } from './utils';
+
+task('populate-descriptor-v3.', 'Populates the descriptor with color palettes and Noun parts')
```

Comment:
@EricDevito why do we need this script when we have the original `populate-descriptor` script that you've also changed to use V3 and seems to do the same things?